python/cudf/cudf/core/dataframe.py

```diff
@@ -4605,6 +4604,8 @@ def join(
             column names when avoiding conflicts.
         sort : bool
             Set to True to ensure sorted ordering.
+        method :
+            This parameter is unused.
```

Comment:
Should we remove the `method` param from the `lhs.merge` call a couple lines later?

Reply:
Yes, good catch. Resolved.